pages/[id]/creation.js

```diff
@@ -120,6 +122,12 @@ const Creation = ({ userProfile, accountId }) => {
 		setIsFiltering(false)
 	}
 
+	const onClickDisplay = (typeDisplay) => {
+		setTokens([])
+		fetchCreatorTokens()
+		setDisplay(typeDisplay)
+	}
+
```

Comment:
since ini ngeset tokensnya jadi empty array lagi, berarti ngefetch dari 0. `idNext`, `lowestPriceNext`, `updatedAtNext` sama `hasMore` nitip di update juga ya ke initial valuenya